docs/source/experiment_reporting.rst

```diff
@@ -109,6 +109,21 @@ Here we show the validation loss in the progress bar
         results = {'progress_bar': logs}
         return results
 
+
+----------
+
+Configure console logging
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Lightning logs useful information about the training process and user warnings to the console.
+You can retrieve the Lightning logger and change it to your liking. For example, increase the logging level
+to see fewer messages like so:
+
+.. code-block:: python
+
+    import logging
+    logging.getLogger("lightning").setLevel(logging.ERROR)
```

Comment:
Maybe we should add a reference to either https://docs.python.org/3/library/logging.html#levels that list the different levels or just the general page https://docs.python.org/3/library/logging.html

Reply:
very good!!

Reply:
done, thanks.